### PR TITLE
🔥Added assignment 3

### PR DESCRIPTION
**Pull request by shaksham08**
- Added Basic React element and same with JSX
-  Added React component and same with JSX
- Created Heading Component with Proper Styling 
image

### CODE REVIEW

File: Week2/Assignment3/src/App.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <h1>Hello</h1>
+    <h2>World</h2>
+    <h3>{value}</h3>
+    {NestedHeadingJSX}
+  </div>
+);
+
+// Composition of component
+const MainTitle = () => (
+  <div>
```
Comment (nitishnivedan) [inline Week2/Assignment3/src/App.js:65]: can use fragments instead of div

File: Week2/Assignment3/src/App.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import ReactDOM from "react-dom";
```
Comment (nitishnivedan) [inline Week2/Assignment3/src/App.js:2]: can have an empty line between libraray imports and local file imports for better readibility

File: Week2/Assignment3/src/index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Namaste React Bootcamp | Assignment 3</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div id="root">
+      <!-- Some error occurred -->
+    </div>
+    <script type="module" src="App.js"></script>
```
Comment (ap221882) [inline Week2/Assignment3/src/index.html:14]: Can put this in the head tag. Modules are deferred by default.

File: Week2/Assignment3/src/style.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+/* Just commenting this CSS */
+/* .heading {
+  color: purple;
+  text-align: center;
+  margin-top: 10em;
```
Comment (ap221882) [inline Week2/Assignment3/src/style.css:5]: Can name this file conventionally as index.css as it styles the root component.

**Review by ap221882 — COMMENTED** [review]
Nicely coded. Reviewed.

File: Week2/Assignment3/src/index.html (hunk 3 above)
Comment (shaksham08) [reply Week2/Assignment3/src/index.html:14]: Interesting I didn't know this

**Pull request merged**